EasyPost/Shipment.cs

```diff
@@ -134,6 +134,29 @@ public void GetRates()
             rates = request.Execute<Shipment>().rates;
         }
 
+        /// <summary>
+        ///     Refresh the rates for this Shipment.
+        /// </summary>
+        /// <param name="parameters">Optional dictionary of parameters for the API request.</param>
+        /// <exception cref="ResourceAlreadyCreated">Shipment has not be created server-side yet.</exception>
+        public void RefreshRates(Dictionary<string, object> parameters = null)
+        {
+            if (id == null)
+            {
+                throw new ResourceNotCreated("Shipment must be created before refreshing rates. Try running `shipment.Create()` first.");
```

Comment:
Nice touch, though it looks like the what we do on other methods is simply create the shipment for them via `Create();`, could we do that here to stay consistent?

Reply:
Resolved in 71c3033